tools/export/uvision4.py

```diff
@@ -28,7 +28,7 @@ class Uvision4(Exporter):
     Exporter class for uvision. This class uses project generator.
     """
     # These 2 are currently for exporters backward compatiblity
-    NAME = 'uVision4'
+    NAME = 'uvision'
```

Comment:
yes. I was refering to Exporter.NAME as this has been in place for some time. If it's used anywhere within tools. 

I understand now why it was renamed :-)